static/_redirects

```diff
@@ -100,6 +106,8 @@
 /guides/comparing-runs /guides/runs 301
 /guides/config /guides/sweeps/define-sweep-configuration 301
 /guides/configurations /guides/track/config 301
+/guides/core/* /guides/models/tables/:splat 301
```

Comment:
Can we use `splat` in Hugo?

https://discourse.gohugo.io/t/can-splat-be-used-in-hugo-server-redirects-config/28292

Reply:
That answer was from 2020, but did give me a clue that apparently `:splat` is a Netlify thing, but it looks like Cloudflare supports it too: https://developers.cloudflare.com/pages/configuration/redirects/ I guess I can check it by going to a redirected page un the preview site to see what happens.

Reply:
Unfortunately splat support seems flaky or nonexistent so I took it away again.